runtime/onert/core/CMakeLists.txt

```diff
@@ -18,9 +21,19 @@ target_link_libraries(onert_core PRIVATE nnfw_lib_misc nnfw_lib_cker)
 target_link_libraries(onert_core PRIVATE nnfw_common)
 target_link_libraries(onert_core PRIVATE nnfw_coverage)
 target_link_libraries(onert_core PRIVATE dl ${LIB_PTHREAD})
+
+# Ruy
+nnfw_find_package(Ruy REQUIRED)
```

Comment:
It is not about h5dumper. I moved `nnfw_find_package(Ruy REQUIRED)` adjacent to other ruy stuffs.
I will make another PR if reviewers want.